operator-pipeline-images/Dockerfile

```
@@ -61,6 +61,18 @@ WORKDIR /home/user
 
 COPY ./operator-pipeline-images ./
 
+# install PDM
+RUN pip3 install --upgrade pip setuptools wheel && pip3 install pdm
+# install dependencies in virtual environment
+COPY ./pdm.lock ./pyproject.toml ./README.md ./
+RUN pdm venv create 3.10 && pdm install --no-lock --no-editable \
+        --group operatorcert \
+        --group operatorcert-dev
+
+ENV VIRTUAL_ENV=./.venv
```

Perhaps there's some confusion about relative/absolute path here. In [pyxis](https://gitlab.cee.redhat.com/metadata/pyxis/pyxis/-/blob/master/dockerfiles/Dockerfile.graphql#L23), we use a absolute path, which 
```suggestion
ENV VIRTUAL_ENV=/home/user/.venv
```

---

Updated.